lessons/topics/02 - Functions and Prototype/README.md

```diff
@@ -0,0 +1,31 @@
+# Functions and prototypes
+
+In this topic we are going to talk about the different kind of functions, and explain little by litle how to structure our code by adding functions and prototypes, making our code more modular and recyclable
```

Comment:
I think recyclable can be a little confusing, I recommend to use reusable.

Reply:
yep, indeed